server/src/main/java/io/seata/server/ParameterParser.java

```diff
@@ -34,11 +34,14 @@
     private static final String SERVER_DEFAULT_BIND_IP = "0.0.0.0";
```

Comment:
this constan is no more required.Because 0.0.0.0 is a valid IP for registering to register center.

Reply:
fixed